dist/k8s-dev/ui.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: apicurio-registry-mt-ui
-          image: quay.io/apicurio/apicurio-registry-mt-ui:latest
+          image: quay.io/apicurio/apicurio-registry-mt-ui:hackathon
```

Comment:
@EricWittmann , using `latest` we have issues with the federated module ... any advice on how to fix it properly?

Reply:
ping @EricWittmann